github/lib.test.w

```diff
@@ -0,0 +1,76 @@
+bring fs;
+bring expect;
+bring util;
+bring "./probot/probot.w" as probot;
+let key = fs.readFile("/path/to/private-key.pem");
+
+let handler = inflight (ctx) => {
+  fs.writeFile("/tmp/111", "pull requesdt opened");
```

Comment:
Probably better to use mkdtemp to put this is a unique temp dir

Reply:
It is a debug entry, we will remove it